code/modules/projectiles/projectile/energy.dm

```diff
@@ -170,6 +170,7 @@
 	icon_state = "yellow_laser"
 	light_color = LIGHT_COLOR_YELLOW
 	stamina = 0
+	reflectability = REFLECTABILITY_NEVER
```

Comment:
Please add a witty comment on why not (given this is such a snowflake exception).

Havign said that: given that reflection from the walls no longer null the firer, this might not even be needed anymore?
Yes, it would make carp users reflect the det projectile back, and set warrant on the detective, but that's a fun thing to have happened to you! I don't think preventing that is worth it?